lib/puppet/type/zabbix_host.rb

```diff
@@ -123,6 +136,36 @@ def insync?(is)
     desc 'Whether it is monitored by an proxy or not.'
   end
 
+  newproperty(:tls_connect) do
+    desc 'Connections to host.'
```

Comment:
Minor, but maybe we can add a better description here since that goes into the REFERENCE.md file. Same for `tls_accept`.